sender.go

```diff
@@ -165,16 +170,18 @@ func (s *sender) trySend(ctx context.Context, evt eventer) error {
 		sp.AddAttributes(tab.StringAttribute("he.message_id", str))
 	}
 
-	recvr := func(err error) {
+	recvr := func(err error, recover bool) {
 		duration := s.recoveryBackoff.Duration()
 		tab.For(ctx).Debug("amqp error, delaying " + strconv.FormatInt(int64(duration/time.Millisecond), 10) + " millis: " + err.Error())
 		time.Sleep(duration)
```

Comment:
(not familiar with this code but should we still sleep if 'recover' is false?

Reply:
For now yes, so that we pause for a bit before retrying (eventually we'll make this configurable).